fp/utils.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+
+def filter_optimal_results_skyline_order(_df, _order_list):
```

Comment:
why is this code in this PR? This belongs to the skyline PR, right?

Reply:
Fixed by deleting it. This file should be deleted for the refactored code.